blackjax/mcmc/metrics.py

```diff
@@ -62,7 +62,11 @@ def __call__(
 
 class Scale(Protocol):
     def __call__(
-        self, position: ArrayLikeTree, element: ArrayLikeTree, inv: ArrayLikeTree
+        self,
+        position: ArrayLikeTree,
+        element: ArrayLikeTree,
+        inv: ArrayLikeTree,
+        trans: ArrayLikeTree,
```

Comment:
Let's turn this to required kwarg:
```suggestion
        *,
        inv: bool,
        trans: bool,
```